security/tor/src/opnsense/mvc/app/models/OPNsense/Tor/General.xml

```diff
@@ -99,5 +99,23 @@
       <default>0</default>
       <Required>Y</Required>
     </dns_map_hosts>
+    <client_authentications>
+      <client_auth type="ArrayField">
+        <enabled type="BooleanField">
+          <default>1</default>
+          <Required>Y</Required>
+        </enabled>
+        <onion_service type="TextField">
+          <Required>Y</Required>
+          <default>exampleexample23.onion</default>
```

Comment:
default is a real world sane value, you're using it as a placeholder / indicator. rather have no default than something like that, but everything is set to Required so it cannot be modelled as empty...

Reply:
it must be required as it would not work without the field being set.

Reply:
this is the value of my dev VM:
kmlnhay2lcmahggk.onion

Reply:
this is more of a Required set of data than individual Required values, something we can't solve today. fine.